src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -51,6 +51,7 @@ stop() {
             ../src/mstop.sh $cluster
         done
     fi
+    killall rbd-mirror
```

Comment:
What does `REMOTE` (`-r` option) stand for?  Would this line make more sense in `"$REMOTE" == "false"` branch where the other daemons are stopped?

Reply:
you were right. it should be inside the condition. `-r` is used to provide the URL of the ceph dashboard which is located in a different machine. Used mostly in local testings.